### ISSUE TITLE
The range syntex for IPv6

### ISSUE THREAD

**Issue by darcyllingyan** [openshift/whereabouts-cni#20]
As when I define the ipam  range for IPv6, if I use below format, it always report error, if I use the example "2001::0/116", it works well, so I'm confused what is the rule for IPv6 range defination.
```
"ipam": {
    "type": "whereabouts",
    "range": "3311:10:11:124::0/64"
  }
```
The erro is below:
```
Error assigning IP: k8s create error: IPPool.whereabouts.cni.cncf.io "3311:10:11:124--64" is invalid: metadata.name: Invalid value: "3311:10:11:124--64": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), 
```

Thanks

**Comment by hubv** [comment]
Anyone could help with this case? We are blocking by this issue.

Thanks!

**Comment by dougbtv** [comment]
This is definitely a bug, thanks for the report!

**Comment by dougbtv** [comment]
This is fixed in downstream master (this repo) and should be fixed in the above referenced PR (60).

I'll work on bringing it into downstream release-4.5

**Comment by hubv** [comment]
Hi @dougbtv 

shall you cherry-pick it in upstream master branch?
Could we use it directly after the code merged in release-4.5?

Best Regards!

**Comment by dougbtv** [comment]
@hubv it's merged in upstream master, and I've got the downstream PR here for 4.5 @ https://github.com/openshift/whereabouts-cni/pull/29 -- so it should be available in 4.5.z once the dependent BZ gets verified and we get the `cherry-pick-approved` on #29 

Thanks! :+1:
